core/test_network/src/lib.rs

```diff
@@ -81,71 +75,44 @@ pub trait TestGenesis: Sized {
 impl TestGenesis for GenesisNetwork {
     fn test(submit_genesis: bool) -> Option<Self> {
         let cfg = Configuration::test();
-        let mut genesis = RawGenesisBlock::new(
-            "alice".parse().expect("Valid"),
-            "wonderland".parse().expect("Valid"),
-            get_key_pair().public_key().clone(),
-        );
+
+        // TODO: Fix this somehow. Probably we need to make `kagami` a library.
```

Comment:
I believe we have an issue for this. Can you add issue number in the comment?

Reply:
I haven't found an issue. Created #3253.

However I'm not sure if it will be realy nice solution, because right now we check `genesis.json` in our repo and with direct genesis from kagami we will not. Also `test_network` will be deprecated in future